build/external-dependencies.props

```diff
@@ -342,6 +342,7 @@
       <NoWarn>KRB2004</NoWarn>
       <VariableName>VisualStudio_NewtonsoftJsonPackageVersion</VariableName>
     </ExternalDependency>
+    <ExternalDependency Include="Castle.Core" Version="$(CastleCorePackageVersion)" Source="$(DefaultNuGetFeed)" />
```

Comment:
Looks like you also need to define CastleCorePackageVersion. We keep the versions list in build/dependencies.props